internal/notify/client.go

```diff
@@ -17,56 +17,47 @@ type Template uint8
 
 const (
 	AttorneyInviteEmail Template = iota
-	AttorneyNameChangeEmail
-	CertificateProviderActingDigitallyDetailsChangedNotSeenLpaSMS
-	CertificateProviderActingDigitallyDetailsChangedSeenLpaSMS
+	CertificateProviderActingDigitallyHasNotConfirmedPersonalDetailsLPADetailsChangedPromptSMS
```

Comment:
![image](https://github.com/ministryofjustice/opg-modernising-lpa/assets/146390/907d9492-17a3-4599-88b6-3447c431a118)

Reply:
Hahaha I hated it as I was writing this out but I didn't want to go re-working the whole template approach on this PR. I'll look at a rework to push some of the context into different template types next.